src/schema/nmdc.yaml

```diff
@@ -953,6 +967,24 @@ slots:
     exact_mappings:
       - OBI:0002475
 
+  is_stranded:
+    description: Is the (RNA) library stranded or non-stranded (unstranded).
+    range: boolean
+    comments:
+      - A value of true means the library is stranded, flase means non-stranded.
+  
+  stranded_orientation:
+    description: Lists the strand orientiation for a stranded RNA library preparation.
+    range: StrandedOrientationEnum
+
+
+  #  mass:
+  #    title: mass
+  #    description: A physical quality that inheres in a bearer by virtue of the proportion of the bearer's amount of matter.
+  #    exact_mappings:
+  #      - PATO:0000125
+  #    range: QuantityValue
```

Comment:
I'd prefer this be deleted instead of commented out, but I don't consider it to be a deal-breaker on a "backmerge," given the higher than usual amount of complexity involved in the merging task. It can be deleted in a subsequent commit/PR.

Reply:
believe it or not, there is a function, non-commented out copy of this in `src/schema/core.yaml`!

deleting this commend-out one as suggested